### ISSUE TITLE
PnP Modern Search not returning search results in iOS 15

### ISSUE THREAD

**Issue by Manjunatha123** [microsoft-search/pnp-modern-search#1347]
__Remove the above section before submitting, and removal of the below template will result in the issue being closed.__

**Version used**
4.2.3

**Describe the bug**
PnP modern search is failing on iOS version 15. It is working fine on iOS version 14.8.

**To Reproduce**
1. Open a Search page with pnp modern search v4(v4.2.3) webparts
2. Search for any term or phrase.
3. Observe that the refiners are loading with the count of results for filter
4. Search results are not being returned in the mobile devices iphone/ ipad with iOS 15 in both chrome and safari browser.
5. Search results are being returned correctly in desktop.

**Expected behavior**
PnP V4 returns search results in all the devices and is compatible with the device updates.

**iOS version**
iOS 15. The pnp modern search v4.2.3 works fine with iOS 14.8.
**Screenshots**

**Additional context**

**Comment by sreeparnamaz** [comment]
The same issue is observed in our application which is in production now & impacting search for all the users who has upgraded to iOS 15.

**Comment by KshamaShrey** [comment]
Even we are facing the same issue and this is heavily impacting the application for the users working with ios15

**Comment by Suj5432** [comment]
Search results are not returned in my application as well once users upgrade to iOS 15. When user enters in search bar I don't see any results being returned for iPhone, checked in iPad - same issue is seen in iPad. I hope there is a quick resolution to this since users expect the results to be populated on search and will be misleading.

**Comment by tista-chatterjee** [comment]
Facing similar issues in our application for ios15. Looking for a quick resolution as it is causing great impact.

**Comment by wobba** [comment]
Debugging on devices is not easy, and unless someone wants to address it it can take some time before I have time to look at it myself.

**Comment by jhholm** [comment]
I just tested this with 4.3.0 development build on iPadOS 15.0 on my iPad mini 4. Cannot reproduce right now. Seems to work with search results + refiners + search box combo with a list template.

Unfortunately I don't have an iOS device at hand. I'll update my iPad to the latest 15.x next.

I debugged the page with https://inspect.dev and found this error showing up that do not show up when using Edge / Chromium.

![image](https://user-images.githubusercontent.com/13876124/136078091-53167e8f-e618-4bc1-93b8-a350ae30c4ff.png)

Quite strange. Could be a red herring. This error starts showing after I add an Search Results Web Part. But doesn't seem to cause any issues for me.

**Comment by Manjunatha123** [comment]
Hi @jhholm @wobba 
Apologies for the late notice.
After going thoroughly though the result template we found that the issue was in the template.
In some places the handlebar expression  line break and space which did not give any issues with Android or windows chome or edge browser.
But it did create issue with chome and safari in iOS 15.
We have fixed it from our side.
Thanks for your patience and time
Thanks all 🙂
